examples/homography_estimation.py

```diff
@@ -407,9 +462,43 @@ def run():
 
             itr += 1
 
+        logger.info(
+            "---------------------------------------------------------------"
+            "---------------------------"
+        )
+        logger.info(f"Forward pass took {sum(forward_times)} ms/epoch.")
+        logger.info(f"Forward pass took {sum(forward_mems)/len(forward_mems)} MBs.")
```

Comment:
MBs/epoch? Or does this report peak usage during operation?

Reply:
We don't need MBs/epoch. Yes, we report peak usage during operation.